packages/cli/src/commands/network/info.ts

```diff
@@ -9,7 +9,7 @@ export default class Info extends BaseCommand {
   static flags = {
     ...BaseCommand.flags,
     lastN: Flags.integer({
-      char: 'n',
+      // We cannot use char: 'n' here because it conflicts with the node flag
```

Comment:
hmm odd right?

Reply:
this would be a breaking change technically, and im surprised it is indeed a conflict

Reply:
nvm i see the issue